admin/controllers/import.php

```diff
@@ -129,14 +134,14 @@ private function CsvImport($type, $dbname)
 			$records = array();
 			$row = 1;
 
-			while (($data = fgetcsv($handle, 10000, ';')) !== FALSE) {
+			while (($data = fgetcsv($handle, 10000, $separator, $delimiter)) !== FALSE) {
 				$num = count($data);
 
 				if ($numfields != $num) {
 					$msg .= "<p>".JText::sprintf('COM_JEM_IMPORT_NUMBER_OF_FIELDS_COUNT_ERROR', $num, $row)."</p>\n";
 				} else {
 					// convert from ansi to utf-8 if required
-					if ($convert) {
+					if ($convert == true) {
```

Comment:
should be now
`if ($convert == false) {`
since meaning of $convert has been reversed